tables/de-g0-core.uti

```diff
@@ -184,26 +185,28 @@ always  (?         2356-6-26
 endnum  !          6-235
 midword !          6-235
 always  \s!        0-6-235
-noback always  !          235
+always  !          235
+
+#word ' 6
```

Comment:
Commented out rule should be removed or explained.

Reply:
Removed the commented rule